doc/architecture/decisions/0012-use-wicked-for-multi-step-forms.md

```diff
@@ -0,0 +1,24 @@
+# 12. Use Wicked for multi-step forms
+
+Date: 2019-12-04
+
+## Status
+
+Accepted
+
+## Context
+
+The create activity form is large and complex, with arguable too much for a
```

Comment:
Small thing: We could include the GOV.UK guidance on this as a viable pattern that gives us some degree of confidence that this approach will make the process more manageable. https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page.